flags/state.py

```diff
@@ -15,6 +16,39 @@ def _flag_state(flag_name, **kwargs):
     return None
 
 
+def _set_flag_state(
+    flag_name, state, create_boolean_condition=True, request=None
+):
+    """ A private function to sets a boolean condition to the desired state """
+    flags = get_flags(request=request)
+    flag = flags.get(flag_name)
+    if flag is None:
+        raise KeyError(f"No flag with name {flag_name} exists")
+
+    db_boolean_condition = next(
+        (
+            c
+            for c in flag.conditions
+            if c.condition == "boolean" and getattr(c, "obj", None) is not None
+        ),
+        None,
+    )
+
+    if db_boolean_condition is not None:
+        # We already have a boolean condition
+        boolean_condition_obj = db_boolean_condition.obj
+    elif db_boolean_condition is None and create_boolean_condition:
+        # We can create a boolean condition and we need to.
+        boolean_condition_obj = FlagState.objects.create(
+            name=flag_name, condition="boolean", value="True"
+        )
+    else:
+        raise ValueError(f"Flag {flag_name} does not have a boolean condition")
+
+    boolean_condition_obj.value = state
+    boolean_condition_obj.save()
```

Comment:
The body of this function is lifted from [the code in Wagtail-Flags that enables/disables flags from a button in the UI](https://github.com/cfpb/wagtail-flags/blob/main/wagtailflags/views.py#L71-L97). Once this is merged and released, that code should be replaced with a API calls to the `enable_flag`/`disable_flag` API introduced here.